packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -241,13 +265,55 @@ export const CMReleasesContainer = () => {
         padding={4}
         shadow="tableShadow"
       >
-        <Flex direction="column" alignItems="stretch" gap={4}>
+        <Flex direction="column" alignItems="stretch" gap={3}>
           <Typography variant="sigma" textColor="neutral600" textTransform="uppercase">
             {formatMessage({
               id: 'content-releases.plugin.name',
-              defaultMessage: 'RELEASES',
+              defaultMessage: 'Releases',
             })}
           </Typography>
+          {releases?.map((release) => {
+            return (
+              <Flex
+                key={release.id}
+                direction="column"
+                alignItems="start"
+                borderWidth="1px"
+                borderStyle="solid"
+                borderColor={getReleaseColorVariant(release.action.type, '200')}
+                hasRadius
+              >
+                <Box
+                  paddingTop={3}
+                  paddingBottom={3}
+                  paddingLeft={4}
+                  paddingRight={4}
+                  background={getReleaseColorVariant(release.action.type, '100')}
+                  width="100%"
+                >
```

Comment:
<img width="1214" alt="Screenshot 2023-12-08 at 10 04 12" src="https://github.com/strapi/strapi/assets/8087692/8791cf9a-9231-4e84-8dc9-d2ce9430a445">

the border radius crops the border. I think you need overflow hidden in the parent Flex

Reply:
good catch and thanks for the suggestion